test/pacmacs-test.el

```diff
@@ -126,3 +126,34 @@
 
      (should (null pacmacs--ghosts))
      (should (= 11 (length pacmacs--terrified-ghosts))))))
+
+(ert-deftest pacmacs--align-score-record-nickname-test ()
+  (let ((pacmacs--max-score-nick-size 5)
```

Comment:
Is it okay to change the constant value in such manner inside a unit test? I wasn't aware that elisp bindings are dynamic by default.